tests/escrow/tests/escrow.ts

```diff
@@ -77,31 +91,33 @@ describe("escrow", () => {
 
   it("Initialize escrow", async () => {
     await program.rpc.initializeEscrow(
```

Comment:
Although the tests run with ts-mocha, none of the RPC methods typecheck properly (small chance it's my editor but I doubt it).